packages/admin/cms-admin/src/generator/future/generateGrid.ts

```diff
@@ -481,7 +546,9 @@ export function generateGrid(
         const { data, loading, error } = useQuery<GQL${gqlTypePlural}GridQuery, GQL${gqlTypePlural}GridQueryVariables>(${instanceGqlTypePlural}Query, {
             variables: {
                 ${hasScope ? `scope,` : ""}
-                ${hasFilter ? `filter: gqlFilter,` : ""}
+                filter: { and: [${hasFilter ? `gqlFilter,` : ""} ${
+        hasGridPropFilter({ config, gridQuery, gqlIntrospection }) ? `...(filter ? [filter] : []),` : "" // TODO handle disable GUI-element for filter defined in filter
```

Comment:
Could you please clarify what you mean with this TODO? Why would we disable filters for base filters?

Reply:
e.g. setting base filter with product.price == 100. adding a filter via GUI product.price > 100 wouldn't make any sense. so it should either be shown in GUI to be transparent for the user or hide the filter-option completely. I think the now renamed prop "filter" could also be used to define "pre-selected" filter as you mentioned above. but in this case the prop should be defined a little bit different, because it should probably configurable to define fixed or uncheckable pre-selected filter. (I know we already had this kind of requirement for one of our projects)